package.nls.zh-cn.json

```diff
@@ -23,6 +23,9 @@
   "commands.SAS.runSelected": "运行选定或所有 SAS 代码",
   "commands.SAS.switchProfile": "切换当前连接配置文件",
   "commands.SAS.updateProfile": "更新连接配置文件",
+  "commands.SAS.upload": "Upload",
+  "commands.SAS.uploadFiles": "Upload File(s)",
```

Comment:
Hi @scottdover, this English string does NOT meet our i18n guidelines. Would you update this? Thanks!

Reply:
Resolved in https://github.com/sassoftware/vscode-sas-extension/pull/584/commits/d48bc53defda2f6a917794d2d455d6d619964ca7